README.md

```diff
@@ -2,6 +2,9 @@
 
 [![CircleCI](https://circleci.com/gh/cats-oss/eslint-config-abema.svg?style=svg)](https://circleci.com/gh/cats-oss/workflows/eslint-config-abema)
 
+_NOTE: By some reasons, core product lines have reduced the dependency for this. See [#247](https://github.com/cats-oss/eslint-config-abema/issues/247)_
```

Comment:
I don't see any explicit benefits to intentionally obfuscate the primary target of this configuration, i.e., AbemaTV as the package name indicates, but it's OK.

Reply:
@nodaguti 

> I don't see any explicit benefits to intentionally obfuscate the primary target of this configuration

I'm not sure about how we should call the web application and related repositories which are parts of many services for AbemaTV officially. I just forget a branding convention for its service excluding that it was a bit complex, so I chose words conservatively 😉,

Reply:
I see your point. Well, having said that we would still be able to include the service name like `core product lines developed by AbemaTV` or `core product lines including AbemaTV's web application`.

Reply:
Thank you. I pushed the newer changeset.